src/main/java/spireMapOverhaul/zones/heavenlyClouds/HeavenlyFlightPower.java

```diff
@@ -0,0 +1,83 @@
+package spireMapOverhaul.zones.heavenlyClouds;
+
+import com.evacipated.cardcrawl.mod.stslib.actions.common.StunMonsterAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractSMOPower;
+
+import static spireMapOverhaul.util.Wiz.atb;
+
+public class HeavenlyFlightPower extends AbstractSMOPower {
+    public static final String POWER_ID = SpireAnniversary6Mod.makeID(HeavenlyFlightPower.class.getSimpleName());
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+    public static final int DAMAGE_REDUCTION = 50;
+    private static final float HEIGHT_ADJUSTMENT = 100.0f;
+    private final int initialAmount;
+
+    public HeavenlyFlightPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, AbstractPower.PowerType.BUFF, false, owner, amount);
+        this.loadRegion("flight");
+        this.priority = 50;
+        this.initialAmount = amount;
+        updateDescription();
+    }
+
+    @Override
+    public void onInitialApplication() {
+        owner.drawY += HEIGHT_ADJUSTMENT * Settings.scale;
```

Comment:
This is great, I was wondering if it would make them float up a bit.

Reply:
glad you like it 👍